seed/lib/mcm/mapper.py

```diff
@@ -178,17 +177,57 @@ def _set_default_concat_config(concat):
     return concat
 
 
+def expand_field(field):
+    """
+    take a field from the csv and expand/split on a delimiter and return a list of individual values
+
+    :param field: str, value to parse
+
+    :return: list of individual values after after delimiting
+    """
+
+    if isinstance(field, str) or isinstance(field, unicode):
+        return re.split(",|;|:", field)
+    else:
+        return [field]
+
+
+def expand_rows(row, delimited_field):
```

Comment:
to make this work right, i'm just creating new rows with the delimiter value changing.